code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -24,13 +24,63 @@
 	can_collar = TRUE
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
+	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// Lizards start with a tail
+	var/has_tail = TRUE 
+
+/mob/living/simple_animal/lizard/handle_automated_action()
+	if(!stat && !buckled)
+		if(prob(1))
+			custom_emote(EMOTE_VISIBLE, pick("sticks out its tongue", "wags its tail.", "lies down."))
+
+	if(isturf(loc) && !incapacitated())
```

Comment:
1) dont use incapacitated, use HAS_TRAIT(src, TRAIT_HANDSBLOCKED) 
2) lizards cant be incapacitated

Reply:
I entirely forgot that resting no longer does that on simplemobs